### DISCUSSION TITLE
error: postgres to aws s3, Saved offset is before replication slot's confirmed lsn

### DISCUSSION THREAD

**Discussion by joew-extremenetworks** [airbytehq/airbyte#45096] [General]
Hello, how do I fix the error below when reading cdc from a postgres database in aws and writing to an aws s3 bucket? The source and destination each individually tested successfully. I also confirmed I was able to read from the db from a separate script, as well as read and write to s3. I tried reseting the connection as well. Any help? Thanks.

error: Saved offset is before replication slot's confirmed lsn

reset instructions: https://www.restack.io/docs/airbyte-knowledge-airbyte-data-reset-guide#:~:text=Steps%20to%20Reset%20Your%20Data

**Comment by jkleinkauff** [comment]
We see this happening when coming from a different pg plugin. We recreated the slot wal2json > pgoutput, then reset the connection and try to sync again. This error will be thrown. The solution for us was to recreate the connection.

[a comment by an automated account is not shown]
